src/bindings/python/src/openvino/runtime/opset14/ops.py

```diff
@@ -4,30 +4,48 @@
 
 """Factory functions for ops added to openvino opset14."""
 from functools import partial
-from typing import Literal, Optional, Union
-import logging
-
-import numpy as np
-
-log = logging.getLogger(__name__)
 
 from openvino.runtime import Node
 from openvino.runtime.opset_utils import _get_node_factory
 from openvino.runtime.utils.decorators import nameable_op
-from openvino.runtime.utils.types import (
-    NodeInput,
-    as_node,
-    as_nodes
-)
+from openvino.runtime.utils.types import NodeInput, as_nodes
 
 _get_node_factory_opset14 = partial(_get_node_factory, "opset14")
 
 
 # -------------------------------------------- ops ------------------------------------------------
+@nameable_op
+def convert_promote_types(
+    left_node: NodeInput,
+    right_node: NodeInput,
+    promote_unsafe: bool = False,
+    pytorch_scalar_promotion: bool = False,
+    u64_integer_promotion_target: str = "f32",
+) -> Node:
+    """Return a node performing conversion to common type based on promotion rules.
+
+    :param left_node: input node with type to be promoted to common one.
+    :param right_node: input node with type to be promoted to common one.
+    :param promote_unsafe: Bool attribute whether to allow promotions that might result in bit-widening, precision loss and undefined behaviors.
+    :param pytorch_scalar_promotion: Bool attribute whether to promote scalar input to type provided by non-scalar input when number format is matching.
+    :param u64_integer_promotion_target: Element type attribute to select promotion result when inputs are u64 and signed integer.
+
+    :return: The new node performing ConvertPromoteTypes operation.
+    """
+    inputs = as_nodes(left_node, right_node)
+
+    attributes = {
+        "promote_unsafe": promote_unsafe,
+        "pytorch_scalar_promotion": pytorch_scalar_promotion,
+        "u64_integer_promotion_target": u64_integer_promotion_target,
```

Comment:
```suggestion
        "u64_integer_promotion_target": u64_integer_promotion_target.to_string() if isinstance(u64_integer_promotion_target, ov.Type) else u64_integer_promotion_target,
```
(*) just an idea how to get it working if ov.Type is not accepted by default.

Reply:
This wasn't necessary, ov Type seem to be converted by default. Modified test cases to use both Type and str